packages/components/package.json

```diff
@@ -64,7 +64,7 @@
     "simulant": "^0.2.2",
     "styled-components": "^4.1.3",
     "timezone-support": "^1.5.5",
-    "uuid": "^3.0.1"
+    "uuid": "^7.0.3"
```

Comment:
there was a breaking change for this one if I remember

Reply:
after dive into it. The only breaking change is uuid() do not exists anymore. 
All our code except ine one place use uuid.v4() which is compatible